### ISSUE TITLE
Client Allocation Request for: Kernelogic

### ISSUE THREAD

**Issue by kernelogic** [keyko-io/filecoin-clients-onboarding#83]
# Client Allocation Questions 
Slingshot participant GHTorrent dataset
https://slingshot.filecoin.io/project/5fb99032210d1a14c4b2a7cf

## Core Information
- Name: Kernelogic
- Website / Social Media: Slack @feiya200
- DataCap Requested: 5T
- Addresses to be Notarized:
f3sfopz3rgcq5x5eaps5himeybsy36sisrgl7i37wbfs5jyvib73vncg7znbjk2fgj6kwnbomv6fwclk464zva

-----------------------------

**Comment by kernelogic** [comment]
- I am a personal at home miner located in Vancouver, Canada.
- I started mining filecoin since June 2020 and participated SpaceRace and every Slingshot competition since start. I participated Slingshot phase 1 mid-term showcase. I am actively filing bug reports on github as well as provide help to community members as much as I can on slack.
- I am asking to become a verified client because I am storing curated public dataset "GHTorrent" as a Slingshot competitor. The whole dataset is 18TB and I plan to store it for 3~5 replications.
- I intend to store the data as per Slingshot rules: 3~5 replications, I get my miner list from slingshot channel on slack from all geographical location. I am not associated with or concentrate on any single entity or affiliate.

**Comment by philippbanhardt** [comment]
Hey @kernelogic - thanks for your request. Can you kindly add more information / context here on:

- yourself , e.g. include links to social media / news mentions etc
- your history working on IPFS / Filecoin, e.g. notable contributions to the community, type of engagement shown over time etc
- type, size and source of the data you (and by extension other Discover participants) are looking to store on Filecoin, e.g. public data vs private data etc
- how you intend to store the data on Filecoin, e.g. redundancy, location / miner preferences, etc

Thanks

**Comment by kernelogic** [comment]
- I am a personal at home miner located in Vancouver, Canada. Linkedin: https://www.linkedin.com/in/feiya200/
- I started mining filecoin since June 2020 and participated SpaceRace and every Slingshot competition since start. I participated Slingshot phase 1 mid-term showcase. I am actively filing bug reports on github as well as provide help to community members as much as I can on slack.
- I am asking to become a verified client because I am storing curated public dataset "GHTorrent" as a Slingshot competitor. The whole dataset is 18TB and I plan to store it for 3~5 replications.
- I intend to store the data as per Slingshot rules: 3~5 replications, I get my miner list from slingshot channel on slack from all geographical location. I am not associated with or concentrate on any single entity or affiliate.
- I will provide detailed transaction list for auditing once approved
